test/contrib/bigquery_gcloud_test.py

```diff
@@ -26,19 +26,44 @@
 import os
 
 import luigi
-from luigi.contrib import bigquery
+import unittest
+from luigi.contrib import bigquery, gcs
+
+try:
+    import googleapiclient.errors
+    import oauth2client
+except ImportError:
+    raise unittest.SkipTest('Unable to load googleapiclient module')
 
-from contrib import gcs_test
 from nose.plugins.attrib import attr
+from testfixtures import should_raise
 
-PROJECT_ID = gcs_test.PROJECT_ID
+# In order to run this test, you should set your GCS/BigQuert project/bucket.
+# Unfortunately there's no mock
```

Comment:
It would be exciting to have automatically generated mocks to speed up local testing... In the manner of http://martinfowler.com/bliki/SelfInitializingFake.html or similar...